contrib/src/main/java/com/datatorrent/contrib/enrich/AbstractEnricher.java

```diff
@@ -185,13 +185,18 @@ public void activate(Context context)
     try {
       cacheManager.initialize();
     } catch (IOException e) {
-      throw new RuntimeException("Unable to initialize primary cache", e);
+      throw new RuntimeException("Unable to initialize cache store", e);
```

Comment:
I meant generally a store of the cache, but I see the confusion with 'CacheStore'. I will change it accordingly.